src/js/actions/LibraryActions.js

```diff
@@ -71,6 +72,10 @@ export default {
             properties: ['openDirectory', 'multiSelections']
         }, (folders) => {
             if(folders !== undefined) {
+
+                folders.forEach((folder, index, array) => {
+                    array[index] = fs.realpathSync(folder);
```

Comment:
I see how map is better with concurrency than a single Promise. 
@YurySolovyov So, we should avoid the code you provided and use a Promisify one? Isn't Promisification more appropriate with API/Modules stuffs?